From 0def8b2abfcb922c9b647092394f079d29e299e1 Mon Sep 17 00:00:00 2001
From: Elmo Todurov <elmo_git@todurov.eu>
Date: Thu, 4 Apr 2024 15:51:35 +0300
Subject: [PATCH] handlers: clarify `help list`

---
 handlers.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/handlers.h b/handlers.h
index 04a98270..daae869c 100644
--- a/handlers.h
+++ b/handlers.h
@@ -81,11 +81,11 @@ bool handler__set(globals_t *vars, char **argv, unsigned argc);
 
 #define LIST_SHRTDOC "list currently known matches"
 #define LIST_LONGDOC "usage: list [max_to_print]\n" \
-               "Print currently known matches, along with details about the\n" \
-               "match, such as its type, location, and last known value. The number in\n" \
-               "the left column is the `match-id`, this can be passed to other commands\n" \
+               "Print currently known matches, along with details about the match:\n" \
+               "[match-id] address, region id + offset, region type, value, flags\n" \
+               "The left column is the `match-id`, this can be passed to other commands\n" \
                "such as `set`, `delete`, etc.\n" \
-               "By default `list` prints up to 10k matches, a numerical parameter" \
+               "By default `list` prints up to 10k matches, a numerical parameter\n" \
                "can be given to change this limit.\n" \
                "The flags displayed indicate the possible types of the variable.\n" \
                "Also the region id, an offset and the region type belonging to a match\n" \
